everestrs/everestrs/build.rs

```diff
@@ -24,8 +28,19 @@ fn find_everest_workspace_root() -> PathBuf {
 /// Returns the Libraries path if this is a standalone build of everest-framework or None if it is
 /// not.
 fn find_libs_in_everest_framework(root: &Path) -> Option<Libraries> {
-    let everestrs_sys = root.join("everest-framework/build/everestrs/libeverestrs_sys.a");
-    let framework = root.join("everest-framework/build/lib/libframework.so");
+    let (everestrs_sys, framework) =
+        if let Ok(everest_framework_dir) = env::var("EVEREST_RS_FRAMEWORK_BINARY_LOCATION") {
```

Comment:
If you pass locations by environment variables, why don't you pass the the complete path to `libeverestrs_sys.a` and `libframework.so` directly, this would be more stable.  Note that at least inside cmake you can get the full path to library-targets (using generator expressions: `FRAMEWORK_LIB=$<TARGET_FILE:everest::framework>`)

Reply:
Absolutely, would add yet another environment variable but I guess the cost of that is quite low